### PR TITLE
feat: Quantized models support

### PR DESCRIPTION
**Pull request by Anush008**
## Description

This is Part II of adding support for quantized models and lazy loading required files.

Part I - #199

### CODE REVIEW

File: fastembed/text/onnx_embedding.py [hunk 1]
```diff
@@ -97,6 +106,17 @@
...
+        "model": "nomic-ai/nomic-embed-text-v1.5-Q",
+        "dim": 768,
+        "description": "Quantized 8192 context length english model",
+        "size_in_GB": 0.13,
+        "sources": {
+            "hf": "nomic-ai/nomic-embed-text-v1.5",
+        },
+        "model_file": "onnx/model_quantized.onnx",
     },
     {
```
Comment (Anush008) [inline fastembed/text/onnx_embedding.py:121]: Adds the above quantized model as POC.

**Comment by Ya-shh** [comment]
for this to  pass for my model I have to store it inside onnx directory.

**Comment by Ya-shh** [comment]
I have now  uploaded the onnx directory

File: fastembed/text/onnx_embedding.py [hunk 2]
```diff
@@ -16,6 +16,7 @@
         "sources": {
             "hf": "yashvardhan7/bge-base-en-onnx",
         },
+        "model_file": "model.onnx",
```
Comment (Anush008) [inline fastembed/text/onnx_embedding.py:19]: The path can be configured. So you didn't have to move.

Comment (Anush008) [reply fastembed/text/onnx_embedding.py:19]: Yikes. You moved it.

Comment (Ya-shh) [reply fastembed/text/onnx_embedding.py:19]: but for most of the model , the path will remain as `onnx/model.onnx` .So it is necessary

**Comment by Ya-shh** [comment]
you can re run it now

File: fastembed/text/onnx_embedding.py (hunk 2 above)
Comment (Anush008) [reply fastembed/text/onnx_embedding.py:19]: Conventionally, yes.

Comment (Ya-shh) [inline fastembed/text/onnx_embedding.py:19]: onnx/model.onnx*

**Comment by Anush008** [comment]
I'll probably have to reopen this PR. We've turned this into a messaging thread. HaHa.

**Pull request closed without merging**